compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -607,15 +607,32 @@ internal actual class ComposeWindow : UIViewController {
 
             override fun pointInside(point: CValue<CGPoint>, event: UIEvent?): Boolean =
                 point.useContents {
-                    val hitsInteropView = attachedComposeContext?.scene?.mainOwner?.hitInteropView(
-                        pointerPosition = Offset(
-                            (x * density.density).toFloat(),
-                            (y * density.density).toFloat()
-                        ),
-                        isTouchEvent = true,
-                    ) ?: false
-
-                    !hitsInteropView
+                    val position = Offset(
+                        (x * density.density).toFloat(),
+                        (y * density.density).toFloat()
+                    )
+
+                    // TODO: temporary solution copying control flow from [ComposeScene.processPress]
+                    // there is no fastReverseAny in Kotlin
```

Comment:
To be consistent, let's just use make an internal function `fastReverseAny`, to simplify the code.

fast* functions are implemented only `androidx` project and originally introduced as an optimization for Android target (not sure though if they optimize something on other targets).

Reply:
Actually, it's a stale comment after input from Ivan. `any` doesn't cover all possible control flows anymore.